core/db/entities.py

```diff
@@ -124,6 +124,139 @@ class ProjectProgress(db.Model):
+    start_date = sqla.Column(sqla.DateTime(), nullable=False)
+    end_date = sqla.Column(sqla.DateTime(), nullable=False)
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="direct_fund_state"), nullable=False)
+    pra_or_other = sqla.Column(sqla.Enum(const.PRAEnum, name="direct_fund_pra"), nullable=False)
+    amount = sqla.Column(sqla.Float(), nullable=False)
+
+    # TODO: should we add a constraint to set this <= amount?
+    contractually_committed_amount = sqla.Column(sqla.Float(), nullable=False)
+
+    # TODO: does this unique index look right?
+    # Unique index for data integrity. There can't be multiple direct fund rows for a single project with
+    # the same date range and direct fund metrics.
+    __table_args__ = (
+        sqla.Index(
+            "ix_unique_direct_fund",
+            "project_id",
+            "start_date",
+            "end_date",
+            "state",
+            "pra_or_other",
+            unique=True,
+        ),
+    )
+
+
+class Capital(db.Model):
+    """Stores Capital data for projects"""
+
+    __tablename__ = "capital_data"
+
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
+    project_id = sqla.Column(sqla.String(), sqla.ForeignKey("project_dim.id"), nullable=False)
+    start_date = sqla.Column(sqla.DateTime(), nullable=False)
+    end_date = sqla.Column(sqla.DateTime(), nullable=False)
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="capital_state"), nullable=False)
+    amount = sqla.Column(sqla.Float(), nullable=False)
+
+    # TODO: does this unique index look right?
+    # Unique index for data integrity. There can't be multiple capital data rows for a single project with
+    # the same date range and capital metrics.
+    __table_args__ = (
+        sqla.Index(
+            "ix_unique_capital",
+            "project_id",
+            "start_date",
+            "end_date",
+            "state",
+            unique=True,
+        ),
+    )
+
+
+class IndirectFundSecured(db.Model):
+    """Stores Indirect Fund Secured Data for Projects."""
+
+    __tablename__ = "indirect_fund_secured_data"
+
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
+    project_id = sqla.Column(sqla.String(), sqla.ForeignKey("project_dim.id"), nullable=False)
+    start_date = sqla.Column(sqla.DateTime(), nullable=False)
+    end_date = sqla.Column(sqla.DateTime(), nullable=False)
+
+    # TODO: Should this reference entities in organisation model, or is it free-text
+    funding_source_name = sqla.Column(sqla.String(), nullable=False)
+    funding_source_category = sqla.Column(
+        sqla.Enum(const.FundingSourceCategoryEnum, name="indirect_secured_fund_source_category"),
+        nullable=False,
+    )
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="indirect_secured_fund_state"), nullable=False)
```

Comment:
"indirect_secured_fund_source_category" should be ""indirect_fund_secured_source_category""
"indirect_secured_fund_state" should be "indirect_fund_secured_state"